packages/kit/types/config.d.ts

```diff
@@ -52,6 +52,7 @@ export interface Config {
 			dir?: string;
 			emitTypes?: boolean;
 			exports?: {
+				behavior?: 'replace' | 'merge';
```

Comment:
these are probably fine defaults. I wonder if there won't be more complicated cases where you want a mixture or something in which case maybe you'd need a `function` or something along those lines. I'm trying to think of what a good name could be if we added more options here in the future. maybe `mergeStrategy`?

Reply:
A mixture should be available to do if the user chooses `merge` behavior and provides the `include`/`exclude` as well.

Only case I can think of right now are these two,, maybe we should change it to a simple boolean instead if there's isn't any other cases, hoping others could chime in on this too.